app/controllers/users/confirmations_controller.rb

```diff
@@ -2,6 +2,12 @@ module Users
   class ConfirmationsController < Devise::ConfirmationsController
     include ValidEmailParameter
 
+    def create
+      super
+
+      UserMailer.already_confirmed(resource).deliver_later if resource.confirmed?
```

Comment:
I don't know how you (or the rails community) feels about adding comments to code but I think it a comment here would useful.